### ISSUE TITLE
Emodjiango (might turn Scratchmoji soon)

### ISSUE THREAD

**Issue by lopste** [ScratchAddons/ScratchAddons#4097]
### Additional context

I have a Scratch forum topic where we create Scratch-forum-style emojis, and we're trying to make it of all the Unicode emojis.

Since emojis are not supported on the forums, it'd be neat if we could add an addon that replaces the normal emojis with the Scratch ones.

I'm working on a JSON file with most of the codepoints of the current one, and the repository with all the emojis is here: https://github.com/lopste/emodjiango (most recent emojis)

And the scratch forum topic is here: https://scratch.mit.edu/discuss/topic/557083/

**Comment by WorldLanguages** [comment]
This is interesting.
Keep in mind open source emoji drawings already exist, tho 🤔

**Comment by lopste** [comment]
Yes, I know stuff like Twemoji exists, but by actually inserting the emojis as images in the posts (so others can see it) we need to use a Scratch image host. Github, sadly isn't one.

**Comment by RedGuy12** [comment]
have you used the forum image uploader addon

**Comment by WorldLanguages** [comment]
So what's the issue? We can upload all twemojis to Scratch

**Comment by mxmou** [comment]
These are drawn in the same pixelated style as the default emojis though, which is certainly an advantage.

**Comment by kccuber-scratch** [comment]
> I'm working on a JSON file with most of the codepoints of the current one, and the repository with all the emojis is here: https://github.com/lopste/emodjiango (most recent emojis)

Why can't we just use a Github action though? There's already the repository so it's even easier

**Comment by lopste** [comment]
Update: @stickfiregames has made a new website that uses a JSON file with all the Unicode codepoints, emojis in the repo, and emojis listed in Scratch-usable hosts. It also has the emojis creator, but I don't think we need that. 

Basically: It'd be much easier to implement.

**Comment by WorldLanguages** [comment]
@lopste Can you link to that?

**Comment by lopste** [comment]
I think the link to the JSON is here:
https://github.com/lopste/forumoji/blob/main/resources/forumoji.json

**Comment by mybearworld** [comment]
Made a PR for this.
